### ISSUE TITLE
Bug : creating a new subtask add extra swimelane and column if you don't navigate through the "all boards" page before 

### ISSUE THREAD

**Issue by Cawet** [wekan/wekan#3868]
### Problem description

* Note: Please anonymize info, and do not add to this public issue any of your Wekan board URLs, passwords, API tokens etc, do you understand?: Yep
* Did you test in newest Wekan?: no
* For new Wekan install, did you configure root-url correctly so Wekan cards open correctly https://github.com/wekan/wekan/wiki/Settings ?
* Wekan version: 5.28 
* If this is about old version of Wekan, what upgrade problem you have?:
* Operating System: Linux
* Deployment Method(snap/docker/sandstorm/mongodb bundle/source): sandstorm
* Http frontend if any (Caddy, Nginx, Apache, see config examples from Wekan GitHub wiki first): 
* Node Version: 12.22
* MongoDB Version: 4.0.24
* Wekan only works on newest desktop Firefox/Chromium/Chrome/Edge/Chromium Edge and mobile Chrome. What webbrowser version are you using? Firefox 89

### Problem description

The problem is similar to https://github.com/wekan/wekan/issues/2256. I already add my feedback in this one, but I finally think it's not exactly the same and the issue seem to be abandon. Sorry if I finally make a duplicate. 

When you create a new subtask on board wish the subtask options are configured, some extra swimelane and column are created if you don't navigate through the "all boards" page before. 

It happens multiple times in a week, and that's a must have for us and it's a pain to clean everything when that bug occurs.
Anyway, here is how exactly reproduce this bug :

**Initial state**

    We have a "child" and a "parent" board.
    The child board is set as the destination in the parent's subtask parameters.
    The column you want subtasks to fall in, is setted in the child's subtask parameters.

**A : It work as expected if :**

    You go to the parent board, by using the wekan UI (for ex. the top menu bar or the main "all board" page
    You go to the parent board, using the navigation pannel on the very top bar of the dashboard (where all your starring boards are pinned)
    You create some subtask
    -> As a result, the subtask goes in the right column of the child board and no extra swimlanes / columns are created

**B: It creates some extra swimlanes and columns when :**

    Yo land on the parent board directly, because you bookmark it, you open it on a new tab, or you simply reload (f5) the page
    You start creating subtasks
    -> Subtasks are going right into a new auto created swimlane / column in the child board.
    -> Now, even if you navigate as in A, it always create new swimlanes (or sometime, it fall into nothing, where you can't find the task anywhere).
    -> If you set again the right destination column in the child board, and you navigate as in A, it will work again properly.

**Comment by avatar1024** [comment]
I can reproduce this bug with Wekan 5.72.

I created a new board, create a card with subtasks (child board did not exist). All subtask where created in a new board (as expected) but each subtask were placed in a separate swimlanes and in separate columns (not expected)!

So to be clear. I created 4 subtask and Wekan automatically create the new child board with four swimlanes (all called default) and four columns (all called queue) in it. Each subtask was place in the new board as such:
- first subtask in first swimlane first column
- second subtask in second swimlane and second column
- and so on.

**Comment by xet7** [comment]
@avatar1024 

Well, somebody (not me) sent that Subtasks feature long time ago. I did read through that code before merging, but I don't remember any details anymore. I don't know why subtasks are created to separate board, etc. And I don't know what to expect, other than, if some would like to change something, they can send PR.

Unless:
- Someone asks how to implement some feature themselves, and asks info about how something works, I will explain those here at GitHub issues
- Someone has paid me at at Commercial Support and Roadmap to do something, links to those at https://wekan.github.io

**Comment by avatar1024** [comment]
Ha I see, thanks for the info. Indeed it looks like the subtask feature has some oddities. In fact I've noticed there are quite a few bugs/issue opened about it, as well some interesting ideas too :)

**Comment by xet7** [comment]
@avatar1024 

Yes, WeKan is combination of huge amount of features and fixes from 217 contributors, with many years of fixes for corner cases etc. Contributors scratch their own itch, develop those features and fixes they need, just figuring out source code and sending PRs.

**Comment by Neikius** [comment]
Yeah, subtasks are mega-weird. I just started also and a subtask board was created for me. All the swimlanes and lists have the same name though (Default for swimlanes and Backlog iirc for lists since I've renamed them). You would expect the sub-board to be in sync with parent board maybe... seems this part is quite unifinished but it is at least usable.
